pkg/utils/ocm.go

```diff
@@ -213,61 +213,62 @@ func loadOCMConfig() (*Config, error) {
 	return cfg, nil
 }
 
-func CreateConnection() *sdk.Connection {
-	token := os.Getenv("OCM_TOKEN")
-	url := os.Getenv("OCM_URL")
-
-	// Unlikely to be set, but check anyway
-	refresh_token := os.Getenv("OCM_REFRESH_TOKEN")
-
-	ocmConfigError := "Unable to load OCM config\nLogin with 'ocm login' or set OCM_TOKEN and OCM_URL environment variables"
-	ocmInvalidURLError := "Invalid OCM_URL found: %s\nValid URL aliases are: 'production', 'staging', 'integration'"
-
-	connectionBuilder := sdk.NewConnectionBuilder()
+func getOcmConfiguration() (*Config, error) {
+	tokenEnv := os.Getenv("OCM_TOKEN")
+	urlEnv := os.Getenv("OCM_URL")
+	refreshTokenEnv := os.Getenv("OCM_REFRESH_TOKEN") // Unlikely to be set, but check anyway
 
 	config := &Config{}
-	err := error(nil)
+	var err error
 
-	if token == "" || url == "" {
-		// If either token or url are not set, try to load them from the config file
+	// If any of the environment variables 'OCM_TOKEN', 'OCM_REFRESH_TOKEN', 'OCM_URL' are not set,
+	// use the configuration file as base.
+	// We don't want to always load this, because the user might only use environment variables.
+	if tokenEnv == "" || urlEnv == "" || refreshTokenEnv == "" {
```

Comment:
I think ocm-container will fall into this block and result in an unrecoverable error when it fails to load the ocm config because they may not set OCM_REFRESH_TOKEN

Reply:
Changed to check if:
- both tokens are empty
**or**
- url is empty